reconcile/utils/vcs.py

```diff
@@ -157,6 +169,44 @@ def get_commit_sha(
         # assume gitlab by default
         return self._gitlab_instance.get_commit_sha(ref=ref, repo_url=repo_url)
 
+    def get_commits_between(
+        self,
+        repo_url: str,
+        commit_from: str,
+        commit_to: str,
+        auth_code: Optional[HasSecret],
+    ) -> list[Commit]:
+        """
+        Return a list of commits between two commits.
+        Note, that the commit_to is included in the result list, whereas commit_from is not included.
+        """
+        commits: list[Commit] = []
+        if "github.com" in repo_url:
```

Comment:
`in` check is a little tricky, what if my gitlab has something like `https://gitlab.com/xxx/test.github.com`, is our github repo url always starts with `https://github.com/`? Then we could just check

```python
        if repo_url.startswith("https://github.com/"):
```

Reply:
Good point! Fixed :+1: